elementary/monitor/dbt_project/macros/alerts/get_test_alerts.sql

```diff
@@ -27,6 +27,8 @@
         extended_alerts as (
             select 
                 alerts_in_time_limit.alert_id,
+                {# Generate elementary unique id which is used to identify between tests, and set it as unique_id #}
+                coalesce(alerts_in_time_limit.test_unique_id, 'None') || '.' || coalesce(alerts_in_time_limit.column_name, 'None') || '.' || coalesce(alerts_in_time_limit.test_sub_type, 'None') as unique_id,
```

Comment:
Because test id is shared between tests, we need to use elementary unique id (like we have on current test results query) to identify between tests.
We use this unique id to dedup alerts.